source/connect/tls.txt

```diff
@@ -0,0 +1,344 @@
+.. _scala-tls:
+
+========================================
+Configure Transport Layer Security (TLS) 
+========================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: security, authentication, transport layer security, encrypt
+
+Overview
+--------
+
+In this guide, you can learn how to use the Transport Layer Security (TLS)
+protocol to secure your connection to a MongoDB deployment.
+
+When you enable TLS for a connection, the {+driver-short+} performs the following actions:
+
+- Uses TLS to connect to the MongoDB deployment
+- Verifies the deployment's certificate
+- Ensures that the certificate certifies the deployment
```

Comment:
I'm not sure what this means.  Was it copied from somewhere else, and if so, where (I don't see it on the Java Sync driver TLS page)?

Reply:
This intro section is from the [Python](https://www.mongodb.com/docs/languages/python/pymongo-driver/current/connect/tls/) / [C++](https://www.mongodb.com/docs/languages/cpp/cpp-driver/current/connect/tls/) pages, but that third bullet point seems to just repeat the one before it. I'll remove it